src/Date.php

```diff
@@ -29,7 +29,7 @@
  * - Uses either strptime() or DateTime to create a date from a format, depending on the format
  * - Uses strftime formatting for dates www.php.net/manual/en/function.strftime.php
  */
-class Date implements DateTimeInterface
```

Comment:
I did comment about that in IRC but it seems to be missing from the PR. @falexandrou, could you do that please?